src/map/entities/mobentity.h

```diff
@@ -102,6 +102,14 @@ enum BEHAVIOUR : uint16
     BEHAVIOUR_NO_TURN      = 0x400  // mob does not turn to face target
 };
 
+enum DRAWIN : uint16
+{
+    DRAWIN_NONE             = 0x000, // no draw-in (mob can only draw-in by explictly calling the lua drawin function)
+    DRAWIN_NORMAL           = 0x001, // basic draw-in where mob will draw-in battle target when target some distance from mob
+    DRAWIN_TO_FRONT_OF_MOB  = 0x002, // determines if draw-in to the center of mob (bit not set) or front of mob (bit set)
+    DRAWIN_INCLUDE_ALLIANCE = 0x004  // determines if draw-in includes just the battle target (bit not set) or entire alliance (bit set)
```

Comment:
Needs trailing comma (if that's relevant in C++, I don't remember?)

Reply:
> Needs trailing comma (if that's relevant in C++, I don't remember?)

Yep, I have fixed this, thanks for comment.